addons/stock/views/stock_rule_views.xml

```diff
@@ -83,7 +83,7 @@
                                 <div><field name="delay" class="oe_inline"/> days</div>
                             </group>
                         </group>
-                        <field name="push_domain" invisible="action not in ['push', 'pull_push']" widget="domain" options="{'model': 'stock.move'}"/>
+                        <field name="push_domain" invisible="action not in ['push', 'pull_push']" widget="domain" options="{'model': 'stock.move.line'}"/>
```

Comment:
I guess we should do a migration for this. Since the existing domain are not stock.move and are not applicable anymore